internal/testutil/acc/config_formatter.go

```diff
@@ -50,3 +57,184 @@ func sortStringMapKeys(m map[string]string) []string {
 	sort.Strings(keys)
 	return keys
 }
+func sortStringMapKeysAny(m map[string]any) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+var matchFirstCap = regexp.MustCompile("(.)([A-Z][a-z]+)")
+var matchAllCap = regexp.MustCompile("([a-z0-9])([A-Z])")
+
+func ToSnakeCase(str string) string {
+	snake := matchFirstCap.ReplaceAllString(str, "${1}_${2}")
+	snake = matchAllCap.ReplaceAllString(snake, "${1}_${2}")
+	return strings.ToLower(snake)
+}
+
+func ClusterResourceHcl(projectID string, req *ClusterRequest) (configStr, clusterName string, err error) {
```

Comment:
We should move cluster specific methods to a separate file

Reply:
Can I do that in a separate PR? There is already a file named advanced_cluster but it has a 'cluster' resource. So I want to clean it up...

Reply:
@maastha maybe I can move the methods to the `cluster.go` file in the `acc` package?
WDYT?